composer.json

```diff
@@ -19,10 +19,11 @@
     "require": {
         "php": "^5.6 || ^7.0",
         "simple-bus/message-bus": "~3.0",
-        "symfony/http-kernel": "~2.3 || ~3.0",
-        "symfony/dependency-injection": "~2.3 || ~3.0",
-        "symfony/yaml": "~2.3 || ~3.0",
-        "symfony/config": "~2.3 || ~3.0"
+        "symfony/http-kernel": "~2.3 || ~3.0 || ~4.0",
```

Comment:
This should be "~2.7 || ~3.2 || ~4.0" (dropping unsupported Symfony versions)

Reply:
Yes, I know. I wanted to keep this PR minimal. I have in mind multiple PR`s:
- drop support for unsupported Symfony versions
- drop support for old 4.x PHPUnit.

If those should be included in this PR then please let me know to make the necessary changes.

Reply:
If you want to drop unsupported Symfony versions, the lower bound for 3.x should be 3.3 then.

Reply:
Official docs are still listing 3.2 among "maintained versions".

Reply:
Thanks for noticing. That's a mistake then. We'll update the docs accordingly.